usecases.md

```diff
@@ -140,6 +140,74 @@ The table below shows the attributes for three different examples. We assume all
 | Site | Sensor X (America/Denver) | Power Plant Y (Etc/GMT+7) | Aggregate Z (America/Phoenix) |
 
 
+### Probabilistic forecasts
+{: .anchor}
+
+A *probabilistic forecast* in the Solar Forecast Arbiter is one or more
+[forecast evaluation time series](#forecastattrs) that describe part or all of a probability
```

Comment:
`(#forecastattrs)  comprising forecast data points that describe `

The forecast data points describe a probability distribution (different distribution at each time step).

Reply:
I'm not clear on what to delete and what to add. Can you copy/paste/edit the whole paragraph?